outwatch/src/main/scala/outwatch/helpers/NativeHelpers.scala

```diff
@@ -33,8 +33,8 @@ private[outwatch] object NativeHelpers {
   }
 
   @noinline def prependSeq[T](source: js.Array[T], other: collection.Seq[T]): js.Array[T] = if (other.isEmpty) source else other match {
-    case wrappedOther:js.WrappedArray[T] =>
-      if (source.isEmpty) wrappedOther else (wrappedOther: js.Array[T]).concat(source)
+    case wrappedOther:js.WrappedArray[_] =>
```

Comment:
Why was this change needed?

Reply:
yes, else the error is:

```scala
[error] [E1] outwatch/src/main/scala/outwatch/helpers/NativeHelpers.scala
[error]      the type test for scalajs.js.WrappedArray[T] cannot be checked at runtime
[error]      L36:     case wrappedOther:js.WrappedArray[T] =>
[error]                    ^
```

Reply:
Ah yeah. Does it work with the unchecked annotation?
```case wrapperOther:js.WrapperArray[T@unchecked]```